python-lib/ontology_tagger.py

```diff
@@ -197,8 +225,17 @@ def _format_no_category(
         self._column_descriptions = formatter._column_descriptions
         return output_df
 
+    def _set_use_models(self, languages: List[AnyStr]) -> bool:
```

Comment:
Shouldn't that method be in the spacy_tokenizer module?

Reply:
9ac5c09